_docs-sources/intro/overview/intro-to-gruntwork.md

```diff
@@ -0,0 +1,17 @@
+# Introduction to Gruntwork
+
+### What is Gruntwork
+
+Gruntwork is a "DevOps accelerator" designed to make it possible to achieve a world-class DevOps setup based completely on infrastructure-as-code in just a few days.
+
+Gruntwork works best for teams building new infrastructure ("greenfield"), whether from scratch or as part of a migration, however it can also be useful for teams with existing infrastructure ("brownfield").
+
+The core elements of a world-class DevOps setup are described in the [Gruntwork Production Framework](gruntwork-production-framework). The [Gruntwork product suite](products) is designed to help software teams implement the Gruntwork Production Framework.
```

Comment:
I'm not sure these three sentences all warrant separate paragraphs. They all read as one cohesive thought, to me.